src/app/page.tsx

```diff
@@ -56,17 +65,7 @@ const ValueSection = () => {
             {valueSection.ctaText}
           </LinkButton>
```

Comment:
This `LinkButton` is getting stretched to fit the container its in due to the flex being added. We could fix this by putting it inside a new `div`. This should put it back in the correct position.
```tsx
<div>
  <LinkButton href={valueSection.ctaHref} variant="primary">
    {valueSection.ctaText}
  </LinkButton>
</div>
```

Reply:
Oh woops, didn't even notice